argoverse/evaluation/stereo/utils.py

```diff
@@ -5,10 +5,10 @@
 from typing import List
 
 import cv2
-import disparity_interpolation
 import matplotlib.pyplot as plt
 import numpy as np
 import pandas as pd
+from numba import njit
```

Comment:
```suggestion
from numba import njit # type: ignore
```

It looks like the CI is failing:
```
argoverse/evaluation/stereo/utils.py:132: error: Untyped decorator makes function "interpolate_disparity" untyped
```

Looking at the `numba` source suggests to me that the `jit` and `njit` decorators aren't typed in a way `mypy` supports. I included a non-ideal suggestion above. I guess if we are considering using this library more we might have to ignore this globally or find something more robust.

Reply:
Yeah, I'd be ok to locally ignore it for now.